src/Microsoft.DotNet.Interactive.ExtensionLab.Tests/NteractKernelExtensionTests.cs

```diff
@@ -77,7 +78,7 @@ public async Task it_checks_and_load_require()
             using var kernel = new CompositeKernel();
 
             var kernelExtension = new NteractKernelExtension();
-            DataExplorerExtensions.Settings.UseUri("https://a.cdn.url/script.js");
+            NteractDataExplorerExtensions.Settings.UseUri("https://a.cdn.url/script.js");
```

Comment:
Any reason to use a method instead of a setter here?

Also, thoughts on how not to make this state global? Can we close over this in the middleware instead, for example? Is changing it once it's set a use case? Is multiple explorers from different CDNs a use case?

Reply:
the Settings class is gone and configuration is done via the Use* api